scripts/postinstall.sh

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env bash
 
+# interrupt on any error
 set -e
 
 npm run build
 
 if [ "$NODE_ENV" = "circleci" ]; then
   # pre-install API to attempt to get its node_modules cached by circleci
   npm run test:e2e:exec api:install
```

Comment:
This is confusing.

Technically, on CircleCI, before installing `website`, you should install `api` (so it should be in `preinstall`). 
Not sure why there is a reference to `e2e` here. I'd simply invoke this with `npm run api:install` (in the same way that we have `npm run db:migrate`, or `npm run db:copy`.